### ISSUE TITLE
Auditer et corriger les tris pour les différentes colonnes du tableau observatoire

### ISSUE THREAD

**Issue by lucaa** [DISIC/wikidemarches#805]
Aujourd'hui le tri est implémenté, ascendent et descendent pour toutes les colonnes du tableau (#719).
Par contre, pour quelques colonnes il est possible que des régles speciales de tri soient nécessaires pour obtenir un résultat cohérent pour l'utilisateur, notamment pour les colonnes qui ont des valeurs texte ou celles qui présentent un mélange de valeurs text et numériques.
Par exemple, pour la colonne indice de satisfaction, le tri considère les valeurs "-" (n/a) et "à venir" comme plus petites que les valeurs les plus basses du pourcentage. Ainsi, un tri décroissant affiche d'abord ces valeurs et tout à la fin les valeurs numériques. Est-ce le résultat attendu pour un utilisateur?

Cette tâche est pour définir toutes ces régles de tri pour toutes les colonnes et les implémenter.

**Comment by lucaa** [comment]
Pour les valeurs de type texte:
* l'ordre n'est pas importante, il est par contre important que les valeurs identiques soient groupées

Pour les valeurs numérique  (usagers satisfaits, disponibilité et DLNUF), ordre "composé" pour les 2 tri (croissant et decroissant), ainsi: 
* on affiche d'abord les valeurs numériques triées (croissant ou decroissant)
* à la fin, on affiche les autres valeurs, triées (pour garder le regroupement), avec - comme dernière valeur.

**Comment by AnthonyBrunelli** [comment]
(à revoir en interne pour déterminer les règles de tri)

**Comment by AnthonyBrunelli** [comment]
Vu avec Georges, ok pour la règle décrite plus haut (commentaire du 16 janvier).

**Comment by lucaa** [comment]
Suite à #871 et #886 , il faut re-voir le tri de l'observatoire et vérifier ce qu'il reste à corriger et comment.

**Comment by AnthonyBrunelli** [comment]
Si je comprends bien, on a fait des modif' pour les valeurs, donc il faut redéfinir les règles de tri. C'est bien ça @lucaa ?

**Comment by lucaa** [comment]
@AnthonyBrunelli oui, c'est ça. Mais il faut commencer par une vérification, pour voir si le tri courant est satisfaisant et si la règle donnée ci-dessus  (https://github.com/DISIC/wikidemarches/issues/805#issuecomment-575239663 ) est toujours la bonne règle à implémenter.

J'avoue qu'il n'est plus très clair pourquoi j'ai créé une deuxième tâche pour le sujet du tri (la #892)...

**Comment by AnthonyBrunelli** [comment]
@lucaa Alors la règle (#805 (comment) ) était bien celle validée mais elle n'est pas mise en place d'après le test rapide que je viens de faire (on a les tirets en premier notamment).
